.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -43,6 +43,14 @@ assignees: ''
 
 <!-- A description of what you expected to happen. -->
 
+## Acceptance Criteria
+
+<!-- A list of one or more requirements that must be met in order to for -->
```

Comment:
```suggestion
<!-- A list of one or more requirements that must be met in order for -->
```

Reply:
Fixed on all three and re-wrapped the comment in 8776ca5. Thank you!